content/MBC3.md

```diff
@@ -16,6 +18,8 @@ supported now.
 
 ### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
 
+## Registers
```

Comment:
The problem is that the register is mapped to the same address space as above, so the description below applies to the section above as well...

Reply:
Fixed